lib/active_model/serializer.rb

```diff
@@ -2,10 +2,12 @@
 
 module ActiveModel
   class Serializer
+    Error = Class.new(StandardError)
```

Comment:
I added it so that I could raise an AMS-specific failure in the 'deprecated' controller methods, and there wasn't any Error class already defined.  I wanted to make sure when testing, for example, that I'm asserting that I'm raising a Runtime error and that it actually is the one from the method, rather than an accident elsewhere in the code... I could check the exception message, I think.